routes/form.js

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable camelcase */
+const router = require('express').Router();
+const Form = require('../models/formModel');
+
+router.post('/form', async (req, res) => {
+    const {
+        quarter,
```

Comment:
if all of these fields are going directly into the form model do they need extracting like this? Open to thoughts on this one as I'm not sure about the convention/best practice